packages/jupyter-ai-magics/jupyter_ai_magics/magics.py

```diff
@@ -124,20 +124,22 @@ def __init__(self, shell):
         super().__init__(shell)
         self.transcript_openai = []
 
-        # suppress warning when using old OpenAIChat provider
-        warnings.filterwarnings(
-            "ignore",
-            message="You are trying to use a chat model. This way of initializing it is "
-            "no longer supported. Instead, please use: "
-            "`from langchain.chat_models import ChatOpenAI`",
-        )
         # suppress warning when using old Anthropic provider
         warnings.filterwarnings(
             "ignore",
             message="This Anthropic LLM is deprecated. Please use "
             "`from langchain.chat_models import ChatAnthropic` instead",
         )
 
+        # suppress warning about our exception handler
+        warnings.filterwarnings(
+            "ignore",
+            message="IPython detected, but you already "
+            "have a custom exception handler installed. I'll skip installing "
+            "Trio's custom handler, but this means exception groups will not "
+            "show full tracebacks.",
+        )
+
```

Comment:
What is the purpose of this block? Is the new `openai` package emitting this warning?

Reply:
Yes, I'm now seeing this when making any request of an OpenAI model. Our custom exception handler populates the `Err` special global variable.